### DISCUSSION TITLE
Where can I find analysis log for SonarLint in PDSOE?

### DISCUSSION THREAD

**Discussion by Ton-Pe** [Riverside-Software/sonar-openedge#1112] [Q&A] [answered]
We have many files that cause a parser error.  SonarLint in PDSOE shows in console view:
`Parser error -- <filename> -- Check analysis log for details`

Can this this analysis log be found somewhere locally on my machine, or does this "analysis log" mean the log file sonar scanner creates when scanning the whole code base? I have tried to find it locally without any success.

(Using version 9.3.3006 of plugin and OE 12.8).

**Comment by gquerret** [comment, marked as answer]
That's an option that you can enable in the Console view:
![image](https://github.com/Riverside-Software/sonar-openedge/assets/6498723/6714b0ac-873a-4bd3-831f-2ee5931b60ec)

**Reply by Ton-Pe** [reply]
Great, thank you very much! I had not noticed that option.
